src/specfem3D/compute_add_sources_viscoelastic.F90

```diff
@@ -890,6 +891,18 @@ double precision function get_stf_viscoelastic(time_source_dble,isource,it_tmp_e
     case (4)
       ! Gaussian source time function by Meschede et al. (2011)
       stf = comp_source_time_function_gauss_2(time_source_dble,hdur(isource))
+    case (5)
+      ! Brune source time function
+      ! hdur is the rise time
+      ! Frequency parameter:
+      f0=2.0d0*PI/hdur(isource)
```

Comment:
better to add the factor 2 PI in the source time function, similar as done above for the others functions.